azure-pipelines.yml

```diff
@@ -81,7 +86,7 @@ jobs:
         workingDirectory: '$(sdkPath)'
         displayName: 'Format Check'
         failOnStderr: true
-        condition: succeededOrFailed()
+        condition: and(succeededOrFailed(), eq(variables['current_go'], 'true'))
```

Comment:
I had to update this condition, because go1.17 has a new `gofmt` output (adding `//go:build` comments, instead of the "old" `// +build` ones).